### PR TITLE
[*] TEST : Enable PHP 7 tests (failure = allowed)

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Enable PHP 7 tests |
| Type? | improvement |
| Category? | TEST |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | нет |
| How to test? | Errr... we'll see what happens on Travis. |

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,11 +18,15 @@ php:
...
-
+  - 7.0
+    
 matrix:
   fast_finish: true
+  allow_failures:
+    - php: 7.0
 
 before_install:
+  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then sudo cp tests/php7-pool.conf ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.d/; fi
```
Comment (mickaelandrieu) [inline .travis.yml]: :)

Comment (firstred) [inline .travis.yml]: Yeah, I think that was the key to enable PHP7 :wink:
 It somehow does not start without a pool configuration.

Comment (mickaelandrieu) [inline .travis.yml]: For now PHP7 is not supported in php-fpm mode: https://docs.travis-ci.com/user/languages/php#PHP-installation

Comment (firstred) [inline .travis.yml]: Yes, so I heard from @julienbourdeau 
But then I came across this blogpost: https://www.marcus-povey.co.uk/2016/02/16/travisci-with-php-7-on-apache-php-fpm/

Comment (firstred) [inline .travis.yml]: Nice! Got it running: https://travis-ci.org/PrestaShop/PrestaShop/jobs/124520641

Some unit tests fail.

**Comment by mickaelandrieu** [comment]
@firstred you rocks !

**Pull request merged**